### PR TITLE
Airbnb Search bar

### PR DESCRIPTION
**Pull request by yevheniia-biliaieva**
- [DEMO LINK](https://yevheniia-biliaieva.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://yevheniia-biliaieva.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,11 +6,39 @@
...
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+
+    <form
+      class="search-bar search-bar--size--big"
+      data-qa="big"
+    >
+      <input
+        data-qa="keypress"
+        class="search-bar__input "
```
Comment (Andriy-Shymkiv) [inline src/index.html:27]:
```suggestion
        class="search-bar__input"
```

File: src/index.html [hunk 2]
```diff
@@ -6,11 +6,39 @@
...
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+
```
Comment (Andriy-Shymkiv) [inline src/index.html:20]:
```suggestion
```

File: src/index.html (hunk 1 above)
Comment (Andriy-Shymkiv) [reply src/index.html:27]: better to give inputs some unique names

File: src/style.css [hunk 3]
```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:9]: bad idea to reset margin/padding for almost all elements on page
please reset it exactly for elements you need

File: src/style.css [hunk 4]
```diff
@@ -1 +1,77 @@
...
+
+.body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
```
Comment (Andriy-Shymkiv) [inline src/style.css:13]: it is more common to keep it inside `html` tag

File: src/style.css [hunk 5]
```diff
@@ -1 +1,77 @@
...
+}
+
+.search-bar__input {
+  padding: 0;
+  border: 0;
+  outline: none;
+}
+
+.search-bar__input::placeholder {
+  font-family: Avenir, sans-serif;
```
Comment (Andriy-Shymkiv) [inline src/style.css:35]: you already did it above for all items. doesn't make sense to do it twice

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
user should be able to click wherever he want to start typing:
image

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Looks good.

**Pull request closed without merging**